Sources/Fluent/Entity/Entity.swift

```diff
@@ -82,6 +84,10 @@ extension Entity {
     public static var name: String {
         return String(describing: self).lowercased()
     }
+    
+    public static var idType: Schema.Field.KeyType? {
```

Comment:
Reason I proposed an optional there is this would leave it up for the driver to interpret as whatever makes sense as the default primary ID type; Integral primary IDs for example may not be possible for some drivers even. But I can see how that capability still remains available, and beside centralising the assumption it is probably also is clearer for more users of the API too.